Index: libchromiumcontent-47.0.2526.110/script/create-dist
===================================================================
--- libchromiumcontent-47.0.2526.110.orig/script/create-dist
+++ libchromiumcontent-47.0.2526.110/script/create-dist
@@ -246,7 +246,7 @@ def main():
   copy_ffmpeg(target_arch)
   copy_sources()
   generate_licenses()
-  create_zip()
+  #create_zip()
 
 
 def parse_args():
